Formula/p/python@3.8.rb

```diff
@@ -25,6 +25,9 @@ class PythonAT38 < Formula
   # build packages later. Xcode-only systems need different flags.
   pour_bottle? only_if: :clt_installed
 
+  # https://devguide.python.org/versions/#versions
+  disable! date: "2024-10-14", because: :deprecated_upstream
```

Comment:
Deprecate?

Reply:
I've recently learned that adding `disable` at a future date immediately puts it in deprecated status until that date. If you check this PR out and run `brew info python@3.8` it will report that it is deprecated.

Reply:
I see. Seems ok to me. Looks good